ext/civicrm_admin_ui/managed/SavedSearch_Administer_Financial_Accounts.mgd.php

```diff
@@ -0,0 +1,176 @@
+        'saved_search_id.name' => 'Administer_Financial_Accounts',
+        'type' => 'table',
+        'settings' => [
+          'actions' => FALSE,
+          'limit' => 50,
+          'classes' => [
+            'table',
+            'table-striped',
+          ],
+          'pager' => [],
+          'placeholder' => 5,
+          'sort' => [
+            [
+              'financial_account_type_id:label',
+              'ASC',
+            ],
+            [
+              'account_type_code',
+              'ASC',
+            ],
+          ],
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'name',
+              'dataType' => 'String',
+              'label' => E::ts('Name'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'description',
+              'dataType' => 'String',
+              'label' => E::ts('Description'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'accounting_code',
+              'dataType' => 'String',
+              'label' => E::ts('Acctg Code'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'financial_account_type_id:label',
+              'dataType' => 'Integer',
+              'label' => E::ts('Account Type'),
+              'sortable' => TRUE,
+              'rewrite' => '[financial_account_type_id:label] ([account_type_code])',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_deductible',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Deductible'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_reserved',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Reserved'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_default',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Default'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_active',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Enabled'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'size' => 'btn-xs',
+              'links' => [
+                [
+                  'entity' => 'FinancialAccount',
+                  'action' => 'update',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-pencil',
+                  'text' => 'Edit',
```

Comment:
Just like we did to the labels, please add internalisation to the text parameter for the buttons.

Reply:
Agreed and done.